File: graphql-api-java/demo/src/main/java/com/graphql/demo/model/Person.java
package com.graphql.demo.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.Collection;

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "PEOPLE")
public class Person {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(name = "NAME", unique = true)
    private String name;

    @Column(name = "AGE")
    private Integer age;

    @OneToMany(mappedBy = "person")
    private Collection<Playlist> playlists;
    public Person(String name, Integer age) {
        this.name = name;
        this.age = age;
    }
}

File: graphql-api/demo/src/main/java/com/graphql/demo/model/Person.kt
package com.graphql.demo.model

import lombok.Getter
import lombok.NoArgsConstructor
import lombok.Setter
import javax.persistence.*

@Getter
@Setter
@NoArgsConstructor
@Entity
@Table(name = "PEOPLE")
class Person(@field:Column(name = "NAME", unique = true) public val name: String, @field:Column(name = "AGE") private val age: Int) {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    public val id: Long? = null

    @OneToMany(mappedBy = "person")
    public val playlists: Collection<Playlist>? = null
}